build.py

```diff
@@ -264,6 +264,8 @@ def core_cmake_args(components, backends, install_dir):
 
     if platform.system() == 'Windows':
```

Comment:
We should generalize the target platform. Can you replace --jetson-build flag with --target-platform arg that takes either "ubuntu", "windows" or "jetpack". After parsing if it is not specified then call a function to automatically determine it. For windows the test is already elsewhere in this file. For jetson is there a similar test we can do. It would get good to auto-detect as much as possible so that users don't actually need to specify --target-platform. Replace current windows test in all places with a test against FLAGS.target_platform.

Reply:
Done